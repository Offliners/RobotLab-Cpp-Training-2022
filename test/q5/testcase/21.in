(9-4+20*17/18/9*10*19+3*13)=
