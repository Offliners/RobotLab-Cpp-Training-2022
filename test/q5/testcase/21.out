-65
-1
3
